src/rules_exceptions.cc

```diff
@@ -195,7 +195,7 @@ bool RulesExceptions::contains(int a) {
         }
     }
 
-    for (auto z : m_ranges) {
+    for (auto& z : m_ranges) {
```

Comment:
```suggestion
    for (const auto& z : m_ranges) {
```
Please see `cppcheck`'s [warning](https://github.com/owasp-modsecurity/ModSecurity/actions/runs/19540276744/job/55952868889?pr=3468#step:5:3289):
```
warning: src/rules_exceptions.cc,198,style,constVariableReference,Variable 'z' can be declared as reference to const
```

Reply:
Sonarcloud also reports [this](https://sonarcloud.io/project/issues?id=owasp-modsecurity_ModSecurity&pullRequest=3468&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true).